nimble/host/mesh/src/proxy.c

```diff
@@ -808,8 +808,8 @@ void bt_mesh_proxy_gatt_disconnect(void)
 	for (i = 0; i < ARRAY_SIZE(clients); i++) {
 		struct bt_mesh_proxy_client *client = &clients[i];
 
-		if (client->conn_handle && (client->filter_type == WHITELIST ||
-					    client->filter_type == BLACKLIST)) {
+		if ((client->conn_handle != BLE_HS_CONN_HANDLE_NONE) && (client->filter_type == WHITELIST || 
```

Comment:
The line is too long here, otherwise looks good

Reply:
Fixed. Thanks.